src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -193,6 +193,28 @@ bool convert_node_input_precision(const std::shared_ptr<ov::Node>& node,
     return false;
 }
 
+// Part of hotfix from CVS-155745 & CVS-155990
```

Comment:
may be better to give description for this function instead referring CVS tickets.

Reply:
Done